Giselle/Index.js

```diff
@@ -0,0 +1,350 @@
+    return sum;
+    //throw new Error("Not Implemneted");
+}
+
+
+
+//Write a program that asks the user for a number n and gives them the possibility to choose between computing the sum(+) or  computing the multipe (*)  of 1, â€¦, n.
+
+homeWork.prototype.choose_operation = function (n, operation) {
+
+    //code here
+    //throw new Error("Not Implemneted");
+    if (operation === "+") {
+       return this.add_1_to(n);
+    } else if (operation === "*") {
+       var mult = 1;
+       for (var i=1; i<=n; i++){
+	   mult= mult*i;
+       }
+       return mult;
+    } else
+      alert("wrong operation");
+              
+
+    //throw new Error("Not Implemneted");
+
+}
+
+//Write a function that returns the largest element in a list
+homeWork.prototype.get_max_number = function (list) {
+    /* 
+      var list = [1,5,8,1,19,3,8]
+     max here 19
+     */
+    //code here
+    
+    return Math.max.apply(null, list);
+    //return list[2];
+    //throw new Error("Not Implemneted");
+}
+
+//Write function that reverses a list, preferably in place.
+
+homeWork.prototype.reverse_list = function (list) {
+    /*
+     list = ['a',4,8,'r']
+     after reverse_list(list)
+     ['r',8,4,'a']
+     */
+    return list.reverse();
+    //throw new Error("Not Implemneted");
+}
+
+//Write a function that checks whether an element exist in a list.
+
+homeWork.prototype.Is_Exist = function (element, list) {
+    /*
+     element ex:5
+     list = [1,5,8,6,9,7]
+     return true ; 
+     element ex:11
+     list = [1,5,8,6,9,7]
+      return false ;
+     */
+
+    //throw new Error("Not Implemneted");
+    //fruits.includes("Mango");
+    var bol = list.includes(element);
+
+    if (bol) {
+       return element+" exist in the list ["+list+"]";
+    } else
+     return element+" doesn't exist in the list ["+list+"]";
+     // alert("wrong operation");
+
+}
+
+//Write a function that returns the elements on odd positions in a list.
+
+homeWork.prototype.get_odd_position_elements = function (list) {
+    /*
+     list =  [1,5,2,3,4,9,10,12]
+     return 5,3,9,12
+     */
+
+    //throw new Error("Not Implemneted");
+
+    return list.filter((num, i)=>i % 2 == 1);
+    
+}
+
+
+//Write a function that returns the odd elements  in a list.
+
+homeWork.prototype.get_odd_numbers = function (list) {
+    /*
+     list =  [1,5,2,3,4,9,10,12]
+     return 1,5,3,9
+     */
+
+    //throw new Error("Not Implemneted");
+
+    return list.filter((number) => number%2!==0);
+    
+
+}
+
+
+//Write a function that concatenates two lists. [a,b,c], [1,2,3]  To [a,b,c,1,2,3]
+
+homeWork.prototype.list_concatenate = function (list1, list2) {
+
+
+    //throw new Error("Not Implemneted");
+
+    var list3 = list1.concat(",",list2);
+    return list3;
+}
+
+//Write a function that takes a number and returns a list of its digits. So for 2342 it should return [2,3,4,2].
+
+homeWork.prototype.convert_number_to_list = function (number) {
+    //throw new Error("Not Implemneted");
```

Comment:
If you  write  this code by yourself , perfect !!
but if you copy , paste from **stackoverflow**  or other website 
it's not code for you !!
this code is professional  : 
`return Array.from(number.toString()).map(Number);`